libs/datamodel/core/tests/attributes/relations/referential_actions.rs

```diff
@@ -388,7 +388,31 @@ fn on_update_cannot_be_defined_on_the_wrong_side() {
 }
 
 #[test]
-fn referential_actions_without_preview_feature_should_error() {
+fn on_delete_without_preview_feature_should_error() {
+    let dml = indoc! { r#"
+        model A {
+            id Int @id
+            bs B[]
+        }
+
+        model B {
+            id Int @id
+            aId Int
+            a A @relation(fields: [aId], references: [id], onDelete: Restrict)
+        }
+    "#};
+
+    let message = "The relation field `a` on Model `B` must not specify the `onDelete` argument in the @relation attribute without enabling the `referentialActions` preview feature.";
+
+    parse_error(dml).assert_are(&[DatamodelError::new_attribute_validation_error(
+        message,
+        "relation",
+        Span::new(127, 145),
+    )]);
```

Comment:
I'm thinking of starting using snapshot tests there, that would let us know where the span is visually, instead of doing manual entry and guess-work.

Reply:
That would be great.